tools/pythonpkg/duckdb_python.cpp

```diff
@@ -115,6 +115,14 @@ PYBIND11_MODULE(DUCKDB_PYTHON_LIB_NAME, m) {
 	m.def("from_parquet", &DuckDBPyRelation::FromParquet,
 	      "Creates a relation object from the Parquet file in file_name", py::arg("file_name"),
 	      py::arg("binary_as_string"), py::arg("connection") = py::none());
+	m.def("from_parquet", &DuckDBPyRelation::FromParquetGlob,
```

Comment:
Can you please collapse these three methods into one, with optional arguments? Python doesn't handle function overloads very well

Reply:
You mean pushing the overload to C++ instead of Python, right?

Reply:
More or less yes, it should be a single method that has optional extra keyword arguments

Reply:
OK. I'll try